MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -32,6 +39,17 @@ final class FeedPresenter {
 	}
 	
 	func didFinishLoadingFeed(with error: Error) {
+		let errorText = "FEED_VIEW_CONNECTION_ERROR".localizedString()
+		errorView.display(FeedErrorModel(errorText: errorText))
 		loadingView.display(FeedLoadingViewModel(isLoading: false))
 	}
 }
+
+
+private extension String {
```

Comment:
Remove extra line break

```suggestion

private extension String {
```